CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,64 @@
+# Cucumber Community Code of Conduct
+
+Cucumber is used and developed by people from all over the world who have a diverse set of skills, personalities and experiences. This diversity is wonderful, and are committed to ensuring that the Cucumber community is a safe, enjoyable and thriving environment for everyone.
+
+As a community, we expect any participant to act according to the following ground rules. They ensure that everyone within the community is clear about the behaviour that is expected of them, and that newcomers know what kind of environment they can expect to find.
+
+## Enforcing this code of conduct
+
+If you experience or observe a violation of this code of conduct, or have any other concerns, please contact us immediately: contact Matt Wynne ([Email](mods@cucumber.io)). We are there to help you. Your reports will be taken seriously, treated in confidence and will not be dismissed or argued with.
```

Comment:
We should either add another person in addition to Matt, or replace "We" with "I". I'd like to be added. Using the single email address is fine, but people should know who'll be reading it.